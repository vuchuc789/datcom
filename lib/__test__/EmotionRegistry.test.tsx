import { render } from '@testing-library/react';
import { EmotionRegistry } from '../EmotionRegistry';

describe('EmotionRegistry', () => {
  test('render correctly', () => {
    const { container } = render(
      <EmotionRegistry>sample content</EmotionRegistry>
    );

    expect(container).toMatchSnapshot();
  });
});

export {};
