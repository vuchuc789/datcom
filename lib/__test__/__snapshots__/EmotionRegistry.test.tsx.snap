// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmotionRegistry render correctly 1`] = `
<div>
  sample content
</div>
`;
